src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1291,38 +1196,44 @@ public static string GetModulePath(string currentProcessModulePath, string hklmM
                 {
                     currentProcessModulePath += hklmMachineModulePath; // += EVT.Machine
                 }
+
+#if !UNIX
+                // Add Windows Modules path
+                string windowsPowerShellModulePath = GetExpandedEnvironmentVariable("SystemDrive", EnvironmentVariableTarget.Process) + @"\WINDOWS\system32\WindowsPowerShell\v1.0\Modules";
```

Comment:
Why did you change to not use `GetWindowsPowerShellPSHomeModulePath()`? or use `s_windowsPowerShellPSHomeModulePath` directly?

Reply:
Will change to use `s_windowsPowerShellPSHomeModulePath`.